src/types.ts

```diff
@@ -116,3 +116,23 @@ export enum VotingMode {
 export type ProposalStatus = 'PROPOSED' | 'ACCEPTED' | 'REJECTED' | 'CANCELED' | 'EXECUTED';
 
 export type GraphUri = `graph://${string}`;
+
+export type DefaultProperties = {
+  name?: string;
+  description?: string;
+  cover?: string;
```

Comment:
Images in the knowledge graph are actually entities, so when they pass cover here they're passing the id for an image entity.

Reply:
Yeah, that was the idea here. We don't have a function to create an Image, but once we have you get an ID. All IDs are strings and so you would need to pass it in here. Since all IDs are strings I used string as a type. Is that fine or something to fix here?

To avoid confusion we could make all IDs a branded type. What do you think? (that said, probably a good idea to do this in a follow up PR)

Reply:
Gotcha everything works as intended then. What we want to avoid is people passing something that _isn't_ an entity id as cover id. Branded types would help at compile-time but requires that consumers are using TS, know what branded/nominal typing is, and could still be invalid at runtime.

I'm good if we want to add branded types but probably also need some runtime validation to make sure they're actually passing an entity id and not a URL for example.